{
    "containers": {
        "cna": {
            "affected": [
                {
                    "product": "n/a",
                    "vendor": "n/a",
                    "versions": [
                        {
                            "status": "affected",
                            "version": "n/a"
                        }
                    ]
                }
            ],
            "dateAssigned": "2017-12-29T00:00:00",
            "datePublic": "2017-09-09T00:00:00",
            "descriptions": [
                {
                    "lang": "en",
                    "value": "pysaml2 version 4.4.0 and older accept any password when run with python optimizations enabled. This allows attackers to log in as any user without knowing their password."
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "description": "n/a",
                            "lang": "en",
                            "type": "text"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "dateUpdated": "2021-02-26T06:06:17",
                "orgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
                "shortName": "mitre"
            },
            "references": [
                {
                    "name": "[debian-lts-announce] 20180701 [SECURITY] [DLA 1410-1] python-pysaml2 security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "https://lists.debian.org/debian-lts-announce/2018/07/msg00000.html"
                },
                {
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://github.com/rohe/pysaml2/issues/451"
                },
                {
                    "name": "GLSA-201801-11",
                    "tags": [
                        "vendor-advisory",
                        "x_refsource_GENTOO"
                    ],
                    "url": "https://security.gentoo.org/glsa/201801-11"
                },
                {
                    "name": "[debian-lts-announce] 20210226 [SECURITY] [DLA 2577-1] python-pysaml2 security update",
                    "tags": [
                        "mailing-list",
                        "x_refsource_MLIST"
                    ],
                    "url": "https://lists.debian.org/debian-lts-announce/2021/02/msg00038.html"
                }
            ],
            "x_legacyV4Record": {
                "CVE_data_meta": {
                    "ASSIGNER": "cve@mitre.org",
                    "DATE_ASSIGNED": "2017-12-29",
                    "ID": "CVE-2017-1000433",
                    "REQUESTER": "predrag.gruevski@gmail.com",
                    "STATE": "PUBLIC"
                },
                "affects": {
                    "vendor": {
                        "vendor_data": [
                            {
                                "product": {
                                    "product_data": [
                                        {
                                            "product_name": "n/a",
                                            "version": {
                                                "version_data": [
                                                    {
                                                        "version_value": "n/a"
                                                    }
                                                ]
                                            }
                                        }
                                    ]
                                },
                                "vendor_name": "n/a"
                            }
                        ]
                    }
                },
                "data_format": "MITRE",
                "data_type": "CVE",
                "data_version": "4.0",
                "description": {
                    "description_data": [
                        {
                            "lang": "eng",
                            "value": "pysaml2 version 4.4.0 and older accept any password when run with python optimizations enabled. This allows attackers to log in as any user without knowing their password."
                        }
                    ]
                },
                "problemtype": {
                    "problemtype_data": [
                        {
                            "description": [
                                {
                                    "lang": "eng",
                                    "value": "n/a"
                                }
                            ]
                        }
                    ]
                },
                "references": {
                    "reference_data": [
                        {
                            "name": "[debian-lts-announce] 20180701 [SECURITY] [DLA 1410-1] python-pysaml2 security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2018/07/msg00000.html"
                        },
                        {
                            "name": "https://github.com/rohe/pysaml2/issues/451",
                            "refsource": "CONFIRM",
                            "url": "https://github.com/rohe/pysaml2/issues/451"
                        },
                        {
                            "name": "GLSA-201801-11",
                            "refsource": "GENTOO",
                            "url": "https://security.gentoo.org/glsa/201801-11"
                        },
                        {
                            "name": "[debian-lts-announce] 20210226 [SECURITY] [DLA 2577-1] python-pysaml2 security update",
                            "refsource": "MLIST",
                            "url": "https://lists.debian.org/debian-lts-announce/2021/02/msg00038.html"
                        }
                    ]
                }
            }
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "96f1cf3c-f80a-4e0f-a063-99a91e3f1325",
                    "shortName": "secretariat-reference",
                    "dateUpdated": "2024-06-30T21:43:46.575Z"
                },
                "references": [
                    {
                        "name": "[debian-lts-announce] 20180701 [SECURITY] [DLA 1410-1] python-pysaml2 security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2018/07/msg00000.html/DEMO_2024_06_19"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://github.com/rohe/pysaml2/issues/451/DEMO_2024_06_19"
                    },
                    {
                        "name": "GLSA-201801-11",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_GENTOO",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://security.gentoo.org/glsa/201801-11/DEMO_2024_06_19"
                    },
                    {
                        "name": "[debian-lts-announce] 20210226 [SECURITY] [DLA 2577-1] python-pysaml2 security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2021/02/msg00038.html/DEMO_2024_06_19"
                    },
                    {
                        "name": "[debian-lts-announce] 20180701 [SECURITY] [DLA 1410-1] python-pysaml2 security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2018/07/msg00000.html/adp-test-6-28"
                    },
                    {
                        "tags": [
                            "x_refsource_CONFIRM",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://github.com/rohe/pysaml2/issues/451/adp-test-6-28"
                    },
                    {
                        "name": "GLSA-201801-11",
                        "tags": [
                            "vendor-advisory",
                            "x_refsource_GENTOO",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://security.gentoo.org/glsa/201801-11/adp-test-6-28"
                    },
                    {
                        "name": "[debian-lts-announce] 20210226 [SECURITY] [DLA 2577-1] python-pysaml2 security update",
                        "tags": [
                            "mailing-list",
                            "x_refsource_MLIST",
                            "x_cve_prog_secretariat_snapshot",
                            "x_cve_prog_secretariat_snapshot@2024-06-28T20:08:51.933Z"
                        ],
                        "url": "https://lists.debian.org/debian-lts-announce/2021/02/msg00038.html/adp-test-6-28"
                    }
                ]
            }
        ]
    },
    "cveMetadata": {
        "assignerOrgId": "8254265b-2729-46b6-b9e3-3dfca2d5bfca",
        "assignerShortName": "mitre",
        "cveId": "CVE-2017-1000433",
        "datePublished": "2018-01-02T23:00:00",
        "dateReserved": "2018-01-02T00:00:00",
        "dateUpdated": "2024-06-30T21:43:46.575Z",
        "state": "PUBLISHED"
    },
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1"
}